### DISCUSSION TITLE
[6.7.2] failed to create volume

### DISCUSSION THREAD

**Discussion by username-is-already-taken2** [concourse/concourse#6658] [Help & Support]
Hi there, 

I've started to notice that a number of tasks across different pipelines have started to fail recently with following message 
![image](https://user-images.githubusercontent.com/13700297/110806693-357df400-827a-11eb-96c7-660c7a9f39a8.png)
I'll try to do some more digging through the journal to see if I can see anything obvious.

Just thought I would ask in case anyone has come across this before.

Best Regards

Gary

**Comment by chenbh** [comment]
This error comes from the worker, most common reason is that the worker disk is full. You should be able to one of these errors in the worker logs:

- `failed-to-create`
- `failed-to-decode-request`
- `failed-to-generate-handle`
- `could-not-produce-strategy`

**Reply by username-is-already-taken2** [reply]
Thank you, looks to be failed-to-create.  Looking at the disk monitoring graphs it doesn't look to run out of space.  I'll try clearing out the worker tmp location and see how it is tonight
